src/redist/targets/Crossgen.targets

```diff
@@ -1,7 +1,7 @@
 <Project>
-  <!-- Crossgen is currently not supported on the s390x architecture. -->
+  <!-- Crossgen is currently not supported on the s390x, ppc64le architecture. -->
```

Comment:
Is this because we use mono on these architectures? Would we still not support crossgen2 if ppc64le moved to CoreCLR?

Reply:
Yes, we have disabled optimization for platforms ppc64le and s390x because we use mono and currently CoreCLR is not available for these platforms.

Reply:
We will have to evaluate support for crosssgen2 if ppc64le moved to CoreCLR.  The comment could be modified to clarify that if needed.